website/content/projects/2019/spec.md

```diff
@@ -403,9 +404,54 @@ You are welcome to create pull requests that do not affect the master branch, an
 These non-master pull requests must meet the same requirements as a pull request to master, except they only require a single code review from someone on your team. Furthermore, it will be up to you whether to ultimately merge the pull request, but a senior developer may opt to not count the pull request towards your score if it is a trivial pull request (i.e., if it represents a trivial amount of effort, like adding someone’s name to a README). As with pull requests to master, we encourage you to touch base with your senior developer before creating your first pull requests.
 
 
-### Peer evaluations
+### Peer feedback
 
-Peer evaluations will happen at the end of Sprints 2, 3, and 4. In each peer evaluation, each student will have to evaluate two team members. Details on how peer evaluations will be carried out will be posted before the first peer evaluation. 
+At several points throughout the project, each student will provide feedback to two other students in their same team. This is similar to a peer evaluation/review (which are fairly common in large software projects), except the feedback you provide will have no effect on the recipient's grade (whereas in a real project, these evaluations/reviews could affect whether you get a raise, a promotion, etc.). Instead, we will be assessing the feedback you provide to others.
+
+At the end of Sprints 2, 3, and 4, you will be assigned two peers who will be providing you with feedback (you will know each other's identities). Before they provide you with feedback, you must think about what you would like to receive feedback on. At the end of Sprints 2 and 3, you should think about the following: 
+
+> *What is something I'd like to improve or do better in the next sprint?*
```

Comment:
This is a great way to put this to prepare them for the exercise. It might be helpful here to include some examples of things they might like to improve, such as:

- clarity of their code for another reader
- how they communicate with other teammates whose code they consume/who consume their code

One thing I think about a lot myself is "What is my personal brand as a programmer? (In my case, a lot of times, it's rigor). Do I live up to that brand? Where do I think some outside feedback would _help me_ better embody that brand? So for me, some feedback on my use, or decision not to use, sensible defaults in my APIs is a place where an outside opinion is helpful.

I don't know that that phrasing would be productive here, but it's how I think about it, and feel free to co-opt part/all/none of this if it helps :)

Reply:
I'm not quite sure how to put this in terms of specific guidelines/etc. in this context (since it's probably to early for them to think about their 'personal brand'), but I'll definitely keep it in mind!